e2e/errors.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from "@playwright/test";
+import test from "./lib/test";
+
+test.describe("pages open without errors", () => {
+  const testRoutes = [
+    { path: "/", testId: "indexPage" },
+    { path: "/markets", testId: "marketsList" },
+    { path: "/liquidity", testId: "liquidityPage" },
+  ];
+
+  for (const route of testRoutes) {
+    test(`rotue "${route.path}"`, async ({ page, consoleErrors }) => {
+      await page.goto(route.path);
+
+      const element = page.getByTestId(route.testId);
+      await element.waitFor();
+      await page.waitForTimeout(5000);
```

Comment:
Looks like `waitForTimeout` should only be used for debugging as it can create flakey tests https://playwright.dev/docs/api/class-page#page-wait-for-timeout

Is there something else we can use instead?